Cargo.toml

```diff
@@ -33,22 +37,25 @@ blake3 = { package = "iroh-blake3", version = "1.4.5"}
 bytes = { version = "1.7", features = ["serde"] }
 derive_more = { version = "1.0.0", features = ["add", "debug", "deref", "display", "from", "try_into", "into"] }
 ed25519-dalek = { version = "2.0.0", features = ["serde", "rand_core"] }
+getrandom = { version = "0.2", default-features = false }
```

Comment:
Do we need this? I've just looked at `iroh-base` today, which also has that, but removing it seems totally fine. Same with my own `browser-iroh-demo`.
It keeps popping up, but I really don't know why it's needed, given I can compile stuff fine for browsers without it. So I'm genuinely curious :D